hub/apps/how-tos/uno-multiplatform.md

```diff
@@ -0,0 +1,119 @@
+
+Soon enough, you will be able to reap the benefits of this approach, as you can target more platforms with a familiar XAML flavor and the codebase you already have.
+
+To create an Uno Platform app:
+
+1. Create a new C# solution using the **Uno Platform App** template, from Visual Studio's **Start Page**
+2. Choose a base template to take your hello world application multi-platform
+
+    :::image type="content" source="images/hello-world/vsix-new-project-options.png" alt-text="Uno solution template for project startup type":::
+
+3. You can optionally choose to customize your app based on the sections on the left side:
+    - **Framework** allows to choose which `TargetFramework` your app will use. [.NET 7.0](https://dotnet.microsoft.com/en-us/download/dotnet/7.0) is a commonly appropriate choice.
+    - **Platforms** provides a list of platforms your application will support. You can choose to support all platforms, or a subset of them
+    - **Presentation** gives a choice about using MVVM (e.g. [MVVM Toolkit](https://learn.microsoft.com/en-us/dotnet/communitytoolkit/mvvm/)) or Uno Platform's MVUX and Feeds
+    - **Projects** gives the ability to add a Server project for APIs and hosting for the WebAssembly project
+    - **Testing** provides Unit Testing and [UI Testing projects](https://github.com/unoplatform/Uno.UITest)
+    - **Features** provides support for WebAssembly PWA and optional VS Code support files
+    - **Extensions** allows to choose for additional [Uno.Extensions](https://github.com/unoplatform/uno.extensions) to kickstart your app faster
+    - **Application** sets the App ID for relevant platforms, used when publishing on various app stores.
+    - **Theme** gives the ability to change between Fluent and Material themes
+
+4. Click the create button
+
+5. Wait for the projects to be created, and their dependencies to be restored
+
+6. A banner at the top of the editor may ask to reload projects, click **Reload projects**:
+    :::image type="content" source="images/hello-world/vs2022-project-reload.png" alt-text="Visual Studio banner offering to reload your projects to complete changes":::
+
+## Copy your existing WinUI 3 project items
+
+Now that you've generated the functional starting point of your multi-platform WinUI application, you can copy the existing WinUI 3 project items into it.
+
+1. Make sure Visual Studio has your WinUI 3 project open, then copy the following items from the WinUI 3 project to the Uno Platform project:
+    - `MainWindow.xaml`
+    - `MainWindow.xaml.cs`
```

Comment:
You need to be specific about where in the new Uno project these files should be copied. Assuming they've selected the default template, these files should go into the class library, in the Presentation folder